### DISCUSSION TITLE
Starter program with next/react/typescript

### DISCUSSION THREAD

**Discussion by clement91190** [sgenoud/replicad#79] [How To]
Hi, 
I would like to use replicad on a project and I have a few heavy dependencies on next js. I was wondering if someone is using replicad with next. I'm struggling to convert the starter-example app to use next js instead of vite. Curious if someone has a solution. Happy to provide a starter app if I figure it out as well. 
It seems I can't even find something like this for just opencascade.js wondering if there is a reason :)

**Comment by sgenoud** [comment]
I used to make it work with webpack, but it was more complicated and was a pita to maintain. But if you provide a starter app, I'd be happy to try to help (and it would make me discover a bit how next works 😉

**Comment by clement91190** [comment]
Got it working after a few iterations. It's here at the moment : https://github.com/clement91190/replicad_next. Happy to open a PR if you want it on this repo @sgenoud

**Reply by sgenoud** [reply]
First, thanks for the work.

This is great living under your name, you can make a PR in the doc linking to it if you want to! 

I'd be curious to see what you are building with it as well (create a showcase conversation for it!)
